common/ruletypes.h

```diff
@@ -241,6 +241,7 @@ RULE_CATEGORY_END()
 RULE_CATEGORY(Skills)
 RULE_INT(Skills, MaxTrainTradeskills, 21, "Highest level for trading skills that can be learnt by the trainer")
 RULE_BOOL(Skills, UseLimitTradeskillSearchSkillDiff, true, "Enables the limit for the maximum difference between trivial and skill for recipe searches and favorites")
+RULE_BOOL(Skills, TrivialNoFail, false, "Makes all trivial recipes no-fail")
```

Comment:
`TrivialRecipesNoFail` is a bit more verbose and explains what this does better.

Reply:
May as well go more verbose with "TrivialTradeskillCombinesNoFail", which better matches the otehr two Skills rules

Reply:
Works for me. :+1: